pkgs/applications/networking/browsers/qutebrowser/default.nix

```
@@ -82,9 +85,9 @@ python3.pkgs.buildPythonApplication {
   installPhase = ''
     runHook preInstall
 
+    export PYTHONOPTIMIZE=1
     make -f misc/Makefile \
-      PYTHON=${python3.pythonOnBuildForHost.interpreter} \
-      PREFIX=. \
+      PYTHON=${(python3.pythonOnBuildForHost.withPackages (p: [p.pip])).interpreter} \
```

Does it suffice to put pip in `nativeBuildInputs`?

---

Turns out if you use
```suggestion
      PYTHON=${(python3.pythonOnBuildForHost.withPackages (p: [p.setuptools])).interpreter} \
```
here then it is picked up correctly by the Makefile and distutils is no longer needed. That's what I do in https://github.com/NixOS/nixpkgs/pull/326607.